repos/rust-lang/lang-team.toml

```diff
@@ -0,0 +1,11 @@
+org = "rust-lang"
+name = "lang-team"
+description = "Home of the Rust lang team"
+bots = ["rustbot", "rfcbot"]
+
+[access.teams]
+lang = "maintain"
+lang-ops = "write"
+lang-docs = "write"
+types = "write"
+release = "write"
```

Comment:
```suggestion
lang-ops = "maintain"
lang-docs = "maintain
types = "maintain"
release = "maintain"
```

looking at github, I see that these things are maintain...

<img width="751" alt="image" src="https://github.com/rust-lang/team/assets/155238/45effeb4-d858-4c29-93e6-a620c63075fa">

I think at *least* ops needs those, so why not the others too.

Reply:
Ok, changed to `maintain` :)